experimental/dds/tree2/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -99,3 +99,23 @@ export function getStableNodeKey(
 		return brand(id);
 	}
 }
+
+/**
+ * Checks whether or not a given path is parented under the root field.
+ * @param path - the path you want to check.
+ * @returns a boolean on whether or not the provided path is parented under the root field.
+ */
+export function isParentedUnderRootField(path: UpPath): boolean {
```

Comment:
We can make this logic more reusable with a little refactoring. Turn this into "getDetachedSequenceContainingPath" and it will be usable in more cases.

Reply:
Refactored to `getDetachedSequenceContainingPath`